content/09.evolution.md

```diff
@@ -138,9 +97,95 @@ While it has been suggested that more thorough investigation of the origins of C
+
+However, one of species of domesticated agricultural animal is severely affected by SARS-CoV-2: the mink (_Neovison vison_).
+While fur farming has declined significantly since the twentieth century, mink farming is still common in China and some European countries, and mink farms continue to exist in the United States.
+Mink belong to the Mustelidae family within Carnivora.
+SARS-CoV-2 was first reported on mink farms in the Netherlands and Denmark in 2020 [@doi:10.2807/1560-7917.ES.2020.25.23.2001005; @doi:10.3201/eid2702.203794].
+Mink were observed to show symptoms of respiratory infection, with varied severity among individuals [@doi:10.2807/1560-7917.ES.2020.25.23.2001005].
+Dissection revealed lung pathology consistent with interstitial pneumonia [@doi:10.2807/1560-7917.ES.2020.25.23.2001005].
+An analysis of five farms in the United States reported mortality rates between 35 and 55% of adult minks [@doi:10.1371/journal.ppat.1009952].
+Subsequently, mink farms worldwide reported outbreaks of SARS-CoV-2.
+Concerns were amplified when novel variants of SARS-CoV-2 were identified as having emerged on Danish mink farms and spread into the human population [@doi:10.2807/1560-7917.ES.2021.26.5.210009; @doi:10.3201/eid2702.203794; @doi:10.1016/j.celrep.2021.109017; @doi:10.3201/eid2702.203794; @doi:10.3390/v13091683].
+The fact that these variants appeared in mink populations before being observed in humans suggests that mink can indeed serve as a viral reservoir [@doi:10.3201/eid2702.203794].
+Concerns about mink-to-human transmission led to the mass destruction of domesticated mink populations in Europe [@doi:10.1016/j.nmni.2020.100816; @doi:10.1126/science.368.6496.1169].
+Introgression from fur farms into wild populations (i.e., feralization) may have also resulted in the spread of SARS-CoV-2 into wild mink populations [@doi:10.1002/fee.2344; @doi:10.3390/ani11051422].
+Therefore, while the specific zoonotic origin of SARS-CoV-2 may still not be clear, the potential for the virus to take hold in species other than humans has been clearly demonstrated by the mink outbreak.
+
+Finally, some species of zoo animals were also monitored to determine whether they were at risk.
+Several species closely related to humans (i.e., the Great Apes) are threatened with extinction and had been identified through _in silico_ studies as likely to be susceptible to SARS-CoV-2 [@doi:10.1073/pnas.2010146117], and therefore the potential for a virus to target these close relatives presented a major concern.
+In early 2021, three gorillas at the San Diego Zoo Safari Park developed respiratory symptoms that were confirmed to be associated with SARS-CoV-2 [@url:https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2].<!--I can't find which species!-->
```

Comment:
Probably _Gorilla beringei beringei_ 

Here is my links to how I found out haha
https://stories.sandiegozoo.org/zoonooz/wonderful-winston/
https://stories.sandiegozoo.org/2021/01/25/gorillasrecovering/
https://www.microbe.tv/twiv/twiv-745/

Reply:
TWIV strikes again!

```suggestion
In early 2021, three gorillas (_Gorilla beringei beringei_) at the San Diego Zoo Safari Park developed respiratory symptoms that were confirmed to be associated with SARS-CoV-2 [@url:https://www.aphis.usda.gov/aphis/newsroom/stakeholder-info/sa_by_date/sa-2021/sa-01/ca-gorillas-sars-cov-2].
```